### PR TITLE
Refunded Products: Display refunded product cells in refunded products screen

### LINKED ISSUE #1703: Refunded Products List: add Refunded Products & Qty section
Now that the Refunded Products List screen can be navigated to

- Add the refunded products & qty section title and cells to the page

For reference, here is the design for the Refunded Products List screen:
Screen Shot 2020-01-06 at 2 36 28 PM

### PR DESCRIPTION
**Pull request by mindgraffiti**
Part 1 of #1703. 

This PR displays the section title and order item cells for products that have been refunded. 

## Notes 
1. This PR does not update the order item quantities in the Order Details page. Coming soon!
2. This PR does not combine order items into a single cell. Coming soon!
3. This PR does not allow pull-to-refresh on the Refunded Products view controller. Coming soon!

### Screenshots
**Design**
Screen Shot 2020-01-07 at 11 43 13 AM

**Implementation**
iPhone Xs

### To Test

1. On the website, choose an order and create a refund by item. Refund 2 items or more and add a note. (Make it a partial refund by refunding only a few items in the order. Don't change the order status to Refunded.) After creating the refund, go to the app and navigate to the Order Details for the chosen order.
2. On Order Details, the "Refunded Products" section title should appear and a "2 Items" detail cell should be in this section. 
3. Tap the "2 Items" cell and navigate to the Refunded Products detail screen. The title should appear. The back button should not have any words. The section titles and cells should appear.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift [hunk 1]
```diff
@@ -465,12 +467,14 @@ private extension OrderDetailsDataSource {
...
 
-        cell.title = NSLocalizedString("Shipping details", comment: "Shipping title for customer info cell")
+        cell.title = NSLocalizedString("Shipping details",
+                                       comment: "Shipping title for customer info cell")
         cell.name = shippingAddress?.fullNameWithCompany
         cell.address = shippingAddress?.formattedPostalAddress ??
             NSLocalizedString(
                 "No address specified.",
-                comment: "Order details > customer info > shipping details. This is where the address would normally display."
+                comment: "Order details > customer info > shipping details. " +
```
Comment (pmusolino) [inline WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift]: Just curiosity: the sum of two string is taken by our localization script?

File: WooCommerce/Classes/ViewModels/Refunds/RefundedProductsDataSource.swift [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        cell.configure(item: itemViewModel, imageService: imageService)
+    }
+}
+
+
+// MARK: - Lookup products
+//
+private extension RefundedProductsDataSource {
+    func lookUpProduct(by productID: Int64) -> Product? {
+        return products.filter({ $0.productID == productID }).first
```
Comment (pmusolino) [inline WooCommerce/Classes/ViewModels/Refunds/RefundedProductsDataSource.swift]: nit: maybe you can use `products.first(where: { $0.productID == productID })`

File: WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift (hunk 1 above)
Comment (mindgraffiti) [reply WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift]: Yep! Concatenation using `+` works. More details in p77Llu-bLz-p2#comment-14876

Comment (mindgraffiti) [reply WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift]: Yep! Concatenation using `+` works. More details in p77Llu-bLz-p2#comment-14876

File: WooCommerce/Classes/ViewModels/Refunds/RefundedProductsDataSource.swift (hunk 2 above)
Comment (mindgraffiti) [reply WooCommerce/Classes/ViewModels/Refunds/RefundedProductsDataSource.swift]: 🤔  `[filter](https://developer.apple.com/documentation/swift/array/3392999-filter)` is the newer method compared to `[first(where:)](https://developer.apple.com/documentation/swift/array/1848165-first)` , and both are O(n) complexity. Is there something that makes you like `first(where:)` better?

**Pull request merged**